packages/mobile/src/account/Settings.tsx

```diff
@@ -76,36 +91,32 @@ const mapDispatchToProps = {
   setAnalyticsEnabled,
   toggleBackupState,
   devModeTriggerClicked,
+  setLockWithPinEnabled,
+  toggleFornoMode,
+}
+
+interface State {
+  fornoSwitchOffWarning: boolean
+  promptFornoModal: boolean
```

Comment:
nit: `promptFornoModal` state is redundant with the route params and can be removed. It will simplify `componentDidMount` too